modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/maven/MavenDeployableProjectModelBuilder.java

```diff
@@ -145,6 +145,8 @@ public DeployableProjectModel build() {
     File pom = getPomFromFolder(projectFolder);
     Model pomModel = getPomModelFromFile(pom);
 
+    deployableArtifactRepositoryFolder = this.mavenConfiguration.getLocalMavenRepositoryLocation();
```

Comment:
Needed for the domain, because its bundle URI is empty and we need to point it to the repository.